dhcpv4/bsdp/vendor_class_identifier.go

```diff
@@ -0,0 +1,9 @@
+// +build !darwin
+
+package bsdp
+
+// MakeVendorClassIdentifier calls the sysctl syscall on macOS to get the
```

Comment:
the comment should be updated, this doesn't use sysctl and is not called on darwin